events/event-models.js

```diff
@@ -0,0 +1,16 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var eventSchema = new Schema({
+  category : {type: String, enum: ['Career','Community','firstother','firstworkshop','mentor','other','outreath','professor','staff','misc']},
```

Comment:
I haven't been paying too much attention to line length, but this line is really quite long.  Per the style guide, lines shouldn't be more than 80-columns wide: https://google.github.io/styleguide/jsguide.html#formatting-column-limit.  There are some guidelines in the style guide about where line breaks should be, but [ClangFormat](https://clang.llvm.org/docs/ClangFormat.html) can do it for you (just make sure to use the Google style, and check that it doesn't do any other weird formatting stuff).
[ClangFormat Download](http://releases.llvm.org/download.html), [Using ClangFormat for .js](https://github.com/google/closure-library/wiki/Formatting-.js-with-clang-format)